src/main/java/org/fungover/haze/HazeList.java

```diff
@@ -256,7 +289,7 @@ public static int parser(String inputString) {
         }
     }
     @SuppressWarnings("squid:S6204")
-    public static List<String> getValueAsList(String textToSplit) {
+    public static List<String> convertToList(String textToSplit) {
```

Comment:
What happens here? Renaming of the method getValueAsList to convertToList?
Because at the same time method getValueAsList looks like its added right now in this PR?